zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -6433,6 +6433,7 @@ components:
             - startDate
             - endDate
             - elementId
+            - elementName
```

Comment:
🔧 We should note in the API description that this field only works for data created with 8.8 and onwards. Instances from prior versions don't contain this data. This applies to sorting and filtering in this endpoint.